packages/documentation/src/pages/getting-started/common-tasks/test.mdx

```diff
@@ -0,0 +1,47 @@
+# Run tests
+
+## Prerequisite
+
+- [Run or build](/getting-started/common-tasks/run-build) the app
+
+## Test commands
+
+| Test type | Test all | Test one | Requires local build to run| Test target | Modules used |
```

Comment:
This table got a little big- open to feedback on this. 

![Screen Shot 2019-04-18 at 16 13 51](https://user-images.githubusercontent.com/4998130/56394561-f90ffd80-61f4-11e9-88d1-9c47009753ef.png)

Reply:
For the test target on accessibility, you could put that it runs against all pages in the sitemap.